MapboxNavigation/RouteManeuverViewController.swift

```diff
@@ -119,7 +119,11 @@ class RouteManeuverViewController: UIViewController {
     
     public func updateStreetNameForStep() {
         if let name = step?.names?.first {
-            streetLabel.unabridgedText = name
+            if let step = step, let ref = step.codes?.first, ["i", "us"].contains(ref.lowercased().components(separatedBy: " ").first!) {
+                streetLabel.unabridgedText = ref.components(separatedBy: " ").joined(separator: "-")
```

Comment:
Can we try to fall back to the step’s `destinations`, since the shield will already be visible to the right?